test/end-to-end/test_tar_export

```diff
@@ -32,13 +32,13 @@ else
 fi
 
 # Remove tarball if already present
-[ -e *.tar.gz ] && rm *.tar.gz
+rm -f ./*.tar.gz
 
 /src/target/debug/hab pkg export tar "$pkg_ident" 
 
 # Check if tarball is present
 
-if [ -f *tar.gz ] ; then
+if [ "$(find . -maxdepth 1 -type f -name "*.tar.gz")" ] ; then
```

Comment:
[The same issue as before](https://github.com/koalaman/shellcheck/wiki/SC2202#globs-dont-work-as-operands-in---use-a-loop) makes the existing code vulnerable. In this case the fix is checking whether the `find` has any output, as [the default test for `[…]` is `-n`](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html#Bash-Conditional-Expressions) (true if nonzero length)